src/walk_dirs.rs

```diff
@@ -403,4 +423,54 @@ mod test {
             discovered
         );
     }
+
+    #[test]
+    fn test_retrieve_empty_imports_on_ignored_dirs_match() {
```

Comment:
We aren't finding an empty file now, this should probably be named `test_retrieve_nothing_on_ignored... `